source/async-bundle.html

```diff
@@ -152,8 +155,15 @@ <h5 id="status-response---complete">Response - Complete</h5>
   <ul>
     <li>HTTP status of <code>200 OK</code></li>
     <li>
-      A body containing a <a href="bundle.html">Bundle Resource</a> with a type of <code>batch-response</code>. The
-      outcome of the kick-off request SHALL be present as the first entry in the bundle.
+      A body containing a <a href="bundle.html">Bundle Resource</a> with a type of <code>batch-response</code>.
+    </li>
+    <li>The outcome of the kick-off request SHALL be present as the first entry in the bundle. A successful completion
+      of the status request SHOULD be used whenever processing has completed, regardless of whether the underlying
+      interaction has succeeded or failed. For example, if a client asynchronously invokes a hypothetical
+      <code>$example</code> operation but omits a required parameter, the server SHOULD respond with a
+      <code>200 OK</code> on the status request once processing is complete, and SHOULD communicate any
```

Comment:
```suggestion
      <code>200 OK</code> on the status request once processing is complete, and would communicate any
```

Reply:
I know I wrote this myself not 60min ago, but I shouldn't have put conformance language in an example ;-)